From nikai@nikai.net  Wed Jul  7 14:58:58 2010
From: Nicolas Kaiser <nikai@nikai.net>
Date: Sun, 27 Jun 2010 17:27:51 +0200
Subject: USB: drivers/usb/Makefile: conditionally descend to 'early'
To: Greg Kroah-Hartman <gregkh@suse.de>
Message-ID: <20100627172751.32deb6aa@absol.kitzblitz>


Don't descend to the EARLY_PRINTK_DBGP directory
unless it is actually used.

Signed-off-by: Nicolas Kaiser <nikai@nikai.net>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/Makefile |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/usb/Makefile
+++ b/drivers/usb/Makefile
@@ -41,7 +41,7 @@ obj-$(CONFIG_USB_MICROTEK)	+= image/
 obj-$(CONFIG_USB_SERIAL)	+= serial/
 
 obj-$(CONFIG_USB)		+= misc/
-obj-y				+= early/
+obj-$(CONFIG_EARLY_PRINTK_DBGP)	+= early/
 
 obj-$(CONFIG_USB_ATM)		+= atm/
 obj-$(CONFIG_USB_SPEEDTOUCH)	+= atm/
